service/src/main/java/org/cbioportal/service/impl/MolecularDataServiceImpl.java

```diff
@@ -219,6 +222,43 @@ public Integer getNumberOfSamplesInMolecularProfile(String molecularProfileId) {
         return molecularDataList;
     }
 
+    @Override
+    public List<GeneMolecularData> getMolecularDataInMultipleMolecularProfilesByGeneQueries(List<String> molecularProfileIds,
+                                                                                            List<String> sampleIds,
+                                                                                            List<GeneFilterQuery> geneQueries,
+                                                                                            String projection) {
+        // Molecular alterations for all genes in the geneQueries
+        List<Integer> entrezGeneIds = geneQueries.stream().map(q -> q.getEntrezGeneId()).collect(Collectors.toList());
```

Comment:
In general, I prefer method references, as they give a rough idea of the type at that point in the stream. Did you just avoid using one here to stay under 120 chars?
```suggestion
        List<Integer> entrezGeneIds = geneQueries.stream().map(GeneFilterQuery::getEntrezGeneId).collect(Collectors.toList());
```

Reply:
No that was not a consideration at all. I have accepted you code improvement.

Reply:
Yes, updated!